### PR TITLE
Milestone fixes

### PR DESCRIPTION
**Pull request by Mazo**
Few fixes for milestone issues

#342 - Cavern warpgates are disabled for now
#358 - All warpgates are broadcast warpgates now
#372 - Instant action spawn point now respect overrides like z offsets (Hopefully this doesn't break anything else @Fate-JH)

On top of that, an unreported bug is fixed where resource silos weren't showing their values correctly after the server was started. All resource silos also now start with 100% NTU.

### CODE REVIEW

**Comment by Mazo** [comment]
Another fix added for #364

**Comment by Mazo** [comment]
Another fix added for #378

File: pslogin/src/main/scala/WorldSessionActor.scala [hunk 1]
```diff
@@ -4329,55 +4332,59 @@ class WorldSessionActor extends Actor
...
-              deadState = DeadState.Alive //must be set here
-            case _ => //seated in something that is not a vehicle or the vehicle is cargo, in which case we can't move
-              deadState = DeadState.Alive
+        case (true, pos) =>
+          // continent.Id == "c1" || continent.Id == "c2" || continent.Id == "c3" || continent.Id == "c4" || continent.Id == "c5" || continent.Id == "c6" ||
+          if (player.isAlive && (admin || continent.Id == "z8" ||
+            continent.Id == "tzshtr" || continent.Id == "tzcotr" || continent.Id == "tzdrtr" ||
+            continent.Id == "tzshnc" || continent.Id == "tzconc" || continent.Id == "tzdrnc" ||
+            continent.Id == "tzshvs" || continent.Id == "tzcovs" || continent.Id == "tzdrvs")) {
+            deadState = DeadState.Release //cancel movement updates
```
Comment (Fate-JH) [inline pslogin/src/main/scala/WorldSessionActor.scala:4369]: What's all this, then?

Up there with CSRZone too.

File: common/src/main/scala/services/vehicle/support/VehicleRemover.scala [hunk 2]
```diff
@@ -26,7 +27,9 @@ class VehicleRemover extends RemoverActor {
     entry.zone.GUID(vehicleGUID) match {
       case Some(vehicle : Vehicle) if vehicle.HasGUID =>
         val zoneId = entry.zone.Id
-        vehicle.Actor ! Vehicle.PrepareForDeletion()
+        if(vehicle.Actor != ActorRef.noSender) {
+          vehicle.Actor ! Vehicle.PrepareForDeletion()
+        }
```
Comment (Fate-JH) [inline common/src/main/scala/services/vehicle/support/VehicleRemover.scala:32]: I'll allow it, though the bulk of this file will be going out the window.

File: pslogin/src/main/scala/WorldSessionActor.scala (hunk 1 above)
Comment (Mazo) [reply pslogin/src/main/scala/WorldSessionActor.scala:4369]: Ported directly from Soun's PTSv3 branch. I think it's to allow non-admins to warp to certain zones like vr training or old oshur.

File: common/src/main/scala/services/vehicle/support/VehicleRemover.scala (hunk 2 above)
Comment (Mazo) [reply common/src/main/scala/services/vehicle/support/VehicleRemover.scala:32]: Yeah I figured when I saw you were getting rid of VehicleRemover, but at least it should work as a temporary bandaid

**Pull request merged**